tests/Unit/ControlSystem/Systems/Test_Shape.cpp

```diff
@@ -0,0 +1,513 @@
+        (*functions_of_time)[shape_name] = std::make_unique<
+            domain::FunctionsOfTime::PiecewisePolynomial<DerivOrder>>(
+            local_initial_time, initial_shape_func,
+            initial_expiration_times.at(shape_name));
+
+        auto initial_size_func =
+            make_array<DerivOrder + 1, DataVector>(DataVector{1, 0.0});
+        initial_size_func[0][0] = ah_radius;
+        const std::string size_name =
+            ControlErrors::detail::size_name<::ah::ObjectLabel::A>();
+        (*functions_of_time)[size_name] = std::make_unique<
+            domain::FunctionsOfTime::PiecewisePolynomial<DerivOrder>>(
+            local_initial_time, initial_size_func,
+            std::numeric_limits<double>::infinity());
+
+        // Excision sphere radius just needs to be inside the horizon. In an
+        // actual run we'd want the excision surface closer to the horizon
+        return 0.75 * ah_radius;
+      };
+
+  const std::string num_ah_coeffs_str = MakeString{} << num_ah_coeffs;
+
+  const std::string input_options =
+      "Evolution:\n"
+      "  InitialTime: 0.0\n"
+      "DomainCreator:\n"
+      "  FakeCreator:\n"
+      "    NumberOfComponents:\n"
+      "      ShapeA: " +
+      num_ah_coeffs_str +
+      "\n"
+      "ControlSystems:\n"
+      "  WriteDataToDisk: false\n"
+      "  ShapeA:\n"
+      "    Averager:\n"
+      "      AverageTimescaleFraction: 0.25\n"
+      "      Average0thDeriv: true\n"
+      "    Controller:\n"
+      "      UpdateFraction: 0.3\n"
+      "    TimescaleTuner:\n"
+      "      InitialTimescales: 0.5\n"
+      "      MinTimescale: 0.1\n"
+      "      MaxTimescale: 10.\n"
+      "      DecreaseThreshold: 4e-3\n"
+      "      IncreaseThreshold: 1e-3\n"
+      "      IncreaseFactor: 1.01\n"
+      "      DecreaseFactor: 0.98\n"
+      "    ControlError:\n";
+
+  // Separation doesn't matter because we are only using AhA
+  system_helper.setup_control_system_test(initial_time, 15.0, input_options,
+                                          initialize_shape_functions_of_time);
+
+  const DataVector zero_dv{num_ah_coeffs, 0.0};
+  const auto zero_array = make_array<DerivOrder + 1, DataVector>(zero_dv);
+
+  // Allocate now and reuse as we go
+  auto initial_ah_coefs = zero_array;
+  domain::FunctionsOfTime::PiecewisePolynomial<DerivOrder> expected_ah_coefs{};
+
+  const std::string deriv_order_string =
+      "DerivOrder=" + get_output(DerivOrder) + ": ";
+  {
+    INFO(deriv_order_string + "Stationary spherical AH");
+    // Set all coefs back to zero first
+    initial_ah_coefs = zero_array;
+    initial_ah_coefs[0][iter.set(0, 0)()] = ah_radius;
+    expected_ah_coefs = {initial_time, initial_ah_coefs,
+                         std::numeric_limits<double>::infinity()};
+    test_shape_control(generator, make_not_null(&system_helper), initial_time,
+                       final_time, l_max, expected_ah_coefs, deriv_eps,
+                       deriv_eps);
```

Comment:
Most of these calls use `deriv_eps` twice.  Is this intentional?

Reply:
Yes because `deriv_eps` is stricter and for these tests that are stationary or linear, both the function and derivative can be checked using this smaller epsilon.

But I agree, the name is sort of confusing. Maybe I'll rename them to `stricter_eps` and `looser_eps` to avoid tying their meaning to func vs deriv.